centreon/src/Core/Security/Token/Infrastructure/API/DeleteToken/DeleteTokenRoute.yaml

```diff
@@ -0,0 +1,16 @@
+DeleteTokenByUserId:
+  methods: DELETE
+  path: /administration/tokens/{tokenName}/users/{userId}
+  requirements:
+    tokenName: '.*'
```

Comment:
The token cannot be empty ? At least `.+` 
👉🏻 But I guess we can be more precise like for example `\w{20,30}`